src/components/button.js

```diff
@@ -22,8 +22,8 @@
       actionId,
       buttonText,
     } = options;
-
-    const { env, useText } = B;
+    const { loading: parentLoading } = parent || {};
```

Comment:
Why do this with prop drilling if you can achieve this with a triggerEvent?

Reply:
Because the form already passes the loading state to it's children. If you prefer a triggerEvent I can change that.

Reply:
Yes please, that is probably before we had triggerEvent and needs to be changed.